tests/broker-engine/notification.robot

```diff
@@ -0,0 +1,1091 @@
+    Engine Config Set Value In Contacts    0    John_Doe    service_notification_commands    command_notif
+
+    ${start}    Get Current Date
+    Start Broker
+    Start Engine
+
+    ${content}    Create List    INITIAL HOST STATE: host_1;
+    ${result}    Find In Log With Timeout    ${engineLog0}    ${start}    ${content}    60
+    Should Be True
+    ...    ${result}
+    ...    An Initial host state on host_1 should be raised before we can start our external commands.
+
+## Time to set the host to CRITICAL HARD.
+
+    FOR   ${i}    IN RANGE    ${3}
+        Process host check result    host_1    1    host_1 DOWN
+    Sleep    1s
+    END
+
+    FOR    ${i}    IN RANGE    ${4}
+        Schedule Forced HOST CHECK    host_1    ${VarRoot}/lib/centreon-engine/config0/rw/centengine.cmd
+        Sleep    5s
+    END
+
+    ${content}    Create List    HOST ALERT: host_1;
+    ${result}    Find In Log With Timeout    ${engineLog0}    ${start}    ${content}    60
+    Should Be True    ${result}    A message telling that notification is not sent
+
+    Stop Engine
+    Kindly Stop Broker
+
+not8
+    [Documentation]    critical host notification
+    [Tags]    broker    engine    host    unified_sql
+    Config Engine    ${1}    ${1}
+    Engine Config Set Value    0    enable_notifications    1    True
+    Engine Config Set Value    0    execute_host_checks    1    True
+    Engine Config Set Value    0    execute_service_checks    1    True
+    Engine Config Set Value    0    log_notifications    1    True
+    Engine Config Set Value    0    log_level_notifications    trace    True
+    Config Broker    central
+    Config Broker    rrd
+    Config Broker    module    ${1}
+    Broker Config Add Item    module0    bbdo_version    3.0.0
+    Broker Config Add Item    rrd    bbdo_version    3.0.0
+    Broker Config Add Item    central    bbdo_version    3.0.0
+    Broker Config Flush Log    central    0
+    Broker Config Log    central    core    error
+    Broker Config Log    central    tcp    error
+    Broker Config Log    central    sql    trace
+    Config Broker Sql Output    central    unified_sql
+    Config Broker Remove Rrd Output    central
+    Clear Retention
+    Engine Config Add Value    0    cfg_file   ${EtcRoot}/centreon-engine/config0/contacts.cfg
+    Engine Config Add Command
+    ...    0
+    ...    command_notif
+    ...    /usr/bin/true
+    Engine Config Set Value In Hosts    0    host_1    notifications_enabled    1
+    Engine Config Set Value In Hosts    0    host_1    notification_options    d,r
+    Engine Config Set Value In Hosts    0    host_1    notification_period    24x7
+    Engine Config Set Value In Hosts    0    host_1    contacts    John_Doe
+    Engine Config Set Value In Contacts    0    John_Doe    host_notification_commands    command_notif
+    Engine Config Set Value In Contacts    0    John_Doe    service_notification_commands    command_notif
```

Comment:
You can use a list with a FOR loop to configure Engine and Broker.  
You can create keyword and use it anywhere